MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableProtocol.c

```diff
@@ -1947,6 +1947,108 @@ InstallAcpiTableFromHob (
   return Status;
 }
 
+/**
+  This function will get ACPI table from gAcpiTableHobGuid Guid hob and install Acpi table from it.

```

Comment:
Please update the comment to indicate updates that are made to the AcpiTableInstance as well. This function is updating the instance with RSDP and RSDT, these are steps in the constructor that will be skipped if this HOB is available.

Reply:
Updated it.